cmd/neofs-node/object.go

```diff
@@ -138,6 +139,24 @@ func (i *delNetInfo) TombstoneLifetime() (uint64, error) {
 	return i.tsLifetime, nil
 }
 
+type innerRingFetcher struct {
+	sidechain *morphClient.Client
+}
+
+func (n *innerRingFetcher) InnerRingKeys() ([][]byte, error) {
+	keys, err := n.sidechain.NeoFSAlphabetList()
```

Comment:
Shouldn't non-alphabet nodes also be processed in ACL sender classifier (e.g. for audit-related requests)?

Reply:
They should and they does. Side chain stores inner ring (not alphabet) keys in `NeoFSAlphabet` role. Unfortunately we don't have special inner ring role for our side chain in NEO so we have to re-use existing ones :(

Reply:
Oh, I see now, thx!